src/Components/DrillAnimationPage.js

```diff
@@ -25,6 +25,9 @@ const styles = StyleSheet.create({
   DrillAnimationPage: {
     height: '100%',
   },
+  VideoPresentation: {
```

Comment:
IMHO style props should be camelCase, as PascalCase is usually used for Components, Classes and Enums